packages/cli/src/workflows/workflow.service.ts

```diff
@@ -1,8 +1,7 @@
 import Container, { Service } from 'typedi';
 import type { INode, IPinData } from 'n8n-workflow';
 import { NodeApiError, Workflow } from 'n8n-workflow';
-import type { FindManyOptions, FindOptionsSelect, FindOptionsWhere } from 'typeorm';
-import { In, Like } from 'typeorm';
+import type { FindOptionsWhere } from 'typeorm';
```

Comment:
Don't we have to also remove this import? maybe re-exporting the types from typeorm from a types.ts in the repository folder? 🤔

Reply:
Type imports are erased during build, no remnant at runtime.